sites/fast-website/src/app/components/color-swatch/color-swatch.ts

```diff
@@ -0,0 +1,7 @@
+import { attr } from "@microsoft/fast-element";
+import { FASTRadio } from "@microsoft/fast-components";
+
+export class ColorSwatch extends FASTRadio {
+    @attr
```

Comment:
```suggestion
    @attr({attribute: "background-color"})
```
Maybe the above for consistency?

Reply:
I think this one got missed